settings/DevHome.Settings/Views/FeedbackPage.xaml.cs

```diff
@@ -297,7 +297,7 @@ private string GetExtensions()
         var extensionsStr = stringResource.GetLocalized("Settings_Feedback_Extensions") + ": \n";
         foreach (var extension in extensions)
         {
-            extensionsStr += extension.PackageFullName + "\n";
+            extensionsStr += extension.PackageFullName + " (" + extension.ExtensionDisplayName + ")\n";

```

Comment:
Do you know if its mandatory for an extension to provide us with a displayname? I'm wondering if we should check if ExtensionDisplayName is null or empty then then maybe put  a localized  "DisplayName not found"  message in the parenthesis

Reply:
Looks like it's required, won't run without it: 
`DEP0700: Registration of the app failed. [0x80080204] error 0xC00CE020: App manifest validation error: The app manifest must be valid as per schema: Line 91, Column 12, Reason: Required attribute 'DisplayName' is missing.`